vpr/src/device/rr_graph_builder.h

```diff
@@ -40,6 +38,37 @@ class RRGraphBuilder {
     t_rr_graph_storage& node_storage();
     /** @brief Return a writable object for update the fast look-up of rr_node */
     RRSpatialLookup& node_lookup();
+    /** @brief Return a writable object for rr_node_metadata */
+    MetadataStorage<int>& rr_node_metadata();
+    /** @brief Return a writable object for rr_edge_metadata */
```

Comment:
Slight modification on the comments

``/** @brief Return a writable object for the meta data on the edges */``

Reply:
I have updated the comments.